src/resources/material.js

```diff
@@ -67,7 +51,25 @@ Object.assign(MaterialHandler.prototype, {
         });
     },
 
+    _assignParserAndBinderBasedOnSubClass: function (data) {
+        if (data.shaderGraph) //maybe there is a better way to tell if node material path should be taken?
+        {
```

Comment:
bracket should be end of previous line :)

Reply:
I will lint it

Reply:
linted